var/spack/repos/builtin/packages/vtk/package.py

```diff
@@ -103,4 +103,27 @@ def cmake_args(self):
                 '-DCMAKE_CXX_FLAGS=-DGLX_GLXEXT_LEGACY'
             ])
 
+            # VTK 6.1.0 (and possibly earlier) does not use
+            # NETCDF_CXX_ROOT to detect NetCDF C++ bindings, so
+            # NETCDF_CXX_INCLUDE_DIR and NETCDF_CXX_LIBRARY must be
+            # used instead to detect these bindings
+            netcdf_cxx_lib = spec['netcdf-cxx'].libs.joined()
+            cmake_args.extend([
+                '-DNETCDF_CXX_INCLUDE_DIR={0}'.format(
+                    spec['netcdf-cxx'].prefix.lib),
```

Comment:
still `prefix.lib` instead of `prefix.include`

Reply:
Good catch! Should be fixed this time, I hope...